data/applications.yml

```diff
@@ -2,230 +2,230 @@
+  team: "#govuk-platform-health"
   api_docs_url: /apis/content-store.html
 
 - github_repo_name: email-alert-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
   metrics_dashboard_url: https://grafana.publishing.service.gov.uk/dashboard/file/email_alert_api.json?refresh=10s&orgId=1
 
 - github_repo_name: email-alert-service
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: imminence
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: link-checker-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
   api_docs_url: /apis/link-checker-api.html
 
 - github_repo_name: publishing-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
   api_docs_url: /apis/publishing-api.html
 
 - github_repo_name: rummager
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
   api_docs_url: /apis/search-api.html
 
 - github_repo_name: asset-manager
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: router-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: support-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: hmrc-manuals-api
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: mapit
   type: APIs
-  team: "#platform-health"
+  team: "#govuk-platform-health"
   api_docs_url: https://mapit.mysociety.org/docs/
 
 # Support
 
 - github_repo_name: content-performance-manager
   type: Supporting apps
-  team: "#data-informed-content"
+  team: "#govuk-data-informed"
 
 - github_repo_name: content-audit-tool
   type: Supporting apps
-  team: "#data-informed-content"
+  team: "#govuk-data-informed"
 
 - github_repo_name: search-admin
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: signon
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: support
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: authenticating-proxy
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: release
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: router
   type: Supporting apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 # frontend
 
 - github_repo_name: calculators
   type: Frontend apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: calendars
   type: Frontend apps
-  team: "#platform-health"
+  team: "#govuk-platform-health"
 
 - github_repo_name: collections
   type: Frontend apps
-  team: "#taxonomy-and-nav"
+  team: "#govuk-taxonomy"
```

Comment:
should this be #govuk-content-pages ?

Reply:
Maybe, I think different people have different opinions of what applications related (or are "owned") by which people (usually product managers) or teams. 

I think this was talked about over the Firebreak week, or at least I remember someone talking about it, but in general, the whole concept seems to have lost relevance since the appearance of the Platform Health team.